src/HonzaBotner/Controllers/AuthController.cs

```diff
@@ -70,7 +70,7 @@ public async Task<ActionResult> Callback()
 
                 return Page(message, 200);
             }
-            catch (InvalidOperationException e)
+            catch (Exception e)
```

Comment:
I used generic `Exception` to catch all the errors. For example, running `https://localhost:5001/Auth/Callback?code=wtf` showed the error dump without this + `src/HonzaBotner.Services/CvutAuthorizationService.cs` changes.